pkg/fmap/fmap.go

```diff
@@ -127,36 +143,57 @@ func Read(f io.Reader) (*FMap, *Metadata, error) {
 		return nil, nil, err
 	}
 
-	// Check for too many fmaps.
-	if bytes.Count(data, Signature) >= 2 {
-		return nil, nil, errors.New("Found multiple signatures")
-	}
+	// Loop over __FMAP__ occurrences until a valid header is found
+	start := 0
+	ValidFmaps := 0
```

Comment:
Make the local variable start with a lowercase -- that's the convention I usually see in Go (although it doesn't seem to be enforced by the linter):

```suggestion
	validFmaps := 0
```